doc/api/cli.md

```diff
@@ -2294,6 +2294,25 @@ The following environment variables are set when running a script with `--run`:
 * `NODE_RUN_PACKAGE_JSON_PATH`: The path to the `package.json` that is being
   processed.
 
+### `--run-from=<path>`
+
+<!--
+added: REPLACEME
+-->
+
+> Stability 1.0 - Early development
+
+Run a `package.json` script from a specified path to a `package.json` file or
+path to the containing folder of a `package.json` file.
+
+The script is run from the directory of the `package.json` file.
```

Comment:
I think this is not entirely correct, as you can also path a subdirectory.
For example, if the `package.json` is in `/some/path/package.json`, you can use `--run-from=/some/path/deeper/path` and it will still work, right?

Wouldn't it be simpler for everything (documentation, implementation, users), if this only supported a path to a folder and said that it will look for the closest `package.json` in the hierarchy?

Reply:
It does parent traversal path